libs/java/server_common/pom.xml

```diff
@@ -163,6 +163,12 @@
       <artifactId>jakarta.mail</artifactId>
       <version>1.6.4</version>
     </dependency>
+      <dependency>
+          <groupId>com.yahoo.athenz</groupId>
+          <artifactId>athenz-zms-core</artifactId>
+          <version>1.8.51-SNAPSHOT</version>
```

Comment:
this must be    `<version>${project.parent.version}</version>`

Reply:
Fixed.